README.md

```diff
@@ -1,2 +1,154 @@
-# hello-rails-react
-First rails-react project
+# README
```

Comment:
**[OPTIONAL REQUIREMENTS]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Ok! I'll consider that for next projects.